### ISSUE TITLE
make.go: Can't build reactGen

### ISSUE THREAD

**Issue by DanielHeath** [perkeep/perkeep#1102]
```
  $ go run make.go
can't load package: package perkeep.org/vendor/myitcv.io/react/cmd/reactGen: cannot find package "perkeep.org/vendor/myitcv.io/react/cmd/reactGen" in any of:
	/usr/local/Cellar/go/1.10.1/libexec/src/perkeep.org/vendor/myitcv.io/react/cmd/reactGen (from $GOROOT)
	/Volumes/Code/heath-it/camlistore/src/perkeep.org/vendor/myitcv.io/react/cmd/reactGen (from $GOPATH)
Error building reactGen: exit status 1
exit status 1
```
Looks like it's adding 'perkeep.org' before vendor/ which is causing it to not be able to find the package.

**Comment by myitcv** [comment]
Just to confirm, this is against 951e9a45bf2af51932da39ef05f263b59d55b239?

**Comment by DanielHeath** [comment]
951e9a45, yep. Looks like make.go now only works if you checkout the repo to `$GOPATH/src/perkeep.org` (previously I'd not had trouble building elsewhere).

**Comment by myitcv** [comment]
I suspect changing [this line](https://github.com/perkeep/perkeep/blob/951e9a45bf2af51932da39ef05f263b59d55b239/make.go#L615) to:

```go
return buildBin("./vendor/myitcv.io/react/cmd/reactGen")
```

will fix that, but:

1. This suggestion is totally untested :)
2. cc @mpl as to whether we even want to do this

**Comment by bradfitz** [comment]
> Looks like make.go now only works if you checkout the repo to $GOPATH/src/perkeep.org (previously I'd not had trouble building elsewhere).

Yeah, that was an unfortunate regression we forgot about when we removed the tmp/buildpath-* cloning.

But I think we'll live with it for now.

Now that there's an implicit $GOPATH at least, we can just make `make.go` tell the user where the perkeep.org checkout needs to be.
